diagnostics-app/src/app/rma/test-viewer/test-viewer.component.ts

```diff
@@ -58,6 +56,15 @@ export class TestViewerComponent {
     });
   }
 
+  ngAfterViewInit() {
+    // Loading child component in `ngAfterViewInit` should be done in a new
+    // change detection cycle. This can be manually triggered via `setTimeout`
```

Comment:
nit: `change detection cycle, which can be ...`

Reply:
Done.